libs/buttons/sim/buttons.ts

```diff
@@ -55,6 +55,7 @@ namespace pxsim {
     export class CommonButtonState {
         usesButtonAB: boolean = false;
         buttons: CommonButton[];
+        buttonsByPin: Map<CommonButton> = {};
```

Comment:
You can use `{[index: number]: CommonButton}` instead of a `Map` to avoid converting to a string below.